test/Elastic.Apm.Tests/Mocks/TestAgentConfigurationReader.cs

```diff
@@ -64,6 +67,12 @@ public class TestAgentConfigurationReader : AbstractConfigurationReader, IConfig
 		public string Environment => ParseEnvironment(Kv(ConfigConsts.EnvVarNames.Environment, _environment, Origin));
 		public string SecretToken => ParseSecretToken(Kv(ConfigConsts.EnvVarNames.SecretToken, _secretToken, Origin));
 		public bool CaptureHeaders => ParseCaptureHeaders(Kv(ConfigConsts.EnvVarNames.CaptureHeaders, _captureHeaders, Origin));
+
+		public double TransactionSampleRate =>
```

Comment:
You copied `TransactionSampleRate` by mistake - I've pushed a change reverting it.